### PR TITLE
task5/ validator for nums

### PR DESCRIPTION
**Pull request by qnislavnachev**
there is a field where we have to input a double number with clicking button submit it should say to us is it  correct or not - correct num is : max 10 characters before ". " and max 5 character after "."

### CODE REVIEW

**Comment by qnislavnachev** [comment]
this task needs a review

**Comment by tsony-tsonev** [comment]
demo ?

File: validator/validateNum.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var regex = /^[1-9]([0-9]{1,9})?(\.[1-9]{1,5})?$/;
+
+function validateNum(num, output) {
```
Comment (tsony-tsonev) [inline validator/validateNum.js]: not a good name

**Comment by qnislavnachev** [comment]
qnislavnachev.github.io/validator/validator.html , sorry my bad I forgot

**Comment by tsony-tsonev** [comment]
LGTM when you think of better name for the function

**Comment by qnislavnachev** [comment]
yeah I think right now

**Pull request merged**